velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -80,6 +81,14 @@ std::tm getDateTime(Timestamp timestamp, const date::time_zone* timeZone) {
   return dateTime;
 }
 
+FOLLY_ALWAYS_INLINE
+std::tm getDateTime(Date date) {
```

Comment:
nit make date const ?

Reply:
I agree with your suggestion, but I was following the style of Timestamp usage. We can do a single pass over this file in a separate PR.